Internal server error when publish event

>  UnicodeEncodeError: 'ascii' codec can't encode character u'\u0142' in position 4: ordinal not in range(128)
> 2016-08-10T23:01:53.574341+00:00 app[web.1]:     print event.searchable_location_name
> 2016-08-10T23:01:53.648680+00:00 app[web.1]: [2016-08-10 23:01:53 +0000] [17] [ERROR] Error handling request /socket.io/?EIO=3&transport=websocket&sid=24d65baebe424c7ebf16e40c19ac741d
> 2016-08-10T23:01:53.648683+00:00 app[web.1]: Traceback (most recent call last):
> 2016-08-10T23:01:53.648684+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/workers/async.py", line 52, in handle
> 2016-08-10T23:01:53.648685+00:00 app[web.1]:     self.handle_request(listener_name, req, client, addr)
> 2016-08-10T23:01:53.648686+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/workers/async.py", line 112, in handle_request
> 2016-08-10T23:01:53.648686+00:00 app[web.1]:     resp.close()
> 2016-08-10T23:01:53.648687+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/http/wsgi.py", line 418, in close
> 2016-08-10T23:01:53.648688+00:00 app[web.1]:     self.send_headers()
> 2016-08-10T23:01:53.648688+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/http/wsgi.py", line 334, in send_headers
> 2016-08-10T23:01:53.648689+00:00 app[web.1]:     tosend = self.default_headers()
> 2016-08-10T23:01:53.648690+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/http/wsgi.py", line 315, in default_headers
> 2016-08-10T23:01:53.648690+00:00 app[web.1]:     elif self.should_close():
> 2016-08-10T23:01:53.648692+00:00 app[web.1]:     if self.status_code < 200 or self.status_code in (204, 304):
> 2016-08-10T23:01:53.648691+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/http/wsgi.py", line 238, in should_close
> 2016-08-10T23:01:53.648693+00:00 app[web.1]: AttributeError: 'Response' object has no attribute 'status_code'

--- a/app/views/admin/models_views/events.py
+++ b/app/views/admin/models_views/events.py
@@ -115,7 +115,6 @@ class EventsView(BaseView):
     @can_access
     def details_view(self, event_id):
         event = DataGetter.get_event(event_id)
-        print event.searchable_location_name
         checklist = {"": ""}
 
         if fields_not_empty(event,

